README-PT.md

```diff
@@ -5,7 +5,8 @@
 [![Sponsors on Open Collective](https://opencollective.com/ifme/sponsors/badge.svg)](#sponsors)
 [![Awesome Humane Tech](https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/humane-tech-badge.svg?sanitize=true)](https://github.com/humanetech-community/awesome-humane-tech)
 
-README en: [Inglês](https://github.com/ifmeorg/ifme/blob/main/README.md), [Espanhol](https://github.com/ifmeorg/ifme/blob/main/README-ES.md), [Francês](https://github.com/ifmeorg/ifme/blob/main/README-FR.md), [Coreano](https://github.com/ifmeorg/ifme/blob/main/README-KO.md), [Indonésio](https://github.com/ifmeorg/ifme/blob/main/README-ID.md), [Turco](https://github.com/ifmeorg/ifme/blob/main/README-TR.md), [Sinhala](https://github.com/ifmeorg/ifme/blob/main/README-LK.md), [Chinês](https://github.com/ifmeorg/ifme/blob/main/README-CN.md)
+README en: [Inglês](https://github.com/ifmeorg/ifme/blob/main/README.md), [Espanhol](https://github.com/ifmeorg/ifme/blob/main/README-ES.md), [Francês](https://github.com/ifmeorg/ifme/blob/main/README-FR.md), [Coreano](https://github.com/ifmeorg/ifme/blob/main/README-KO.md), [Indonésio](https://github.com/ifmeorg/ifme/blob/main/README-ID.md), [Turco](https://github.com/ifmeorg/ifme/blob/main/README-TR.md), [Sinhala](https://github.com/ifmeorg/ifme/blob/main/README-LK.md), [Chinês](https://github.com/ifmeorg/ifme/blob/main/README-CN.md), 
+[hindi](https://github.com/ifmeorg/ifme/blob/main/README-HI.md)
```

Comment:
Let's capitalize `hindi` to `Hindi` here as well.

Reply:
Thanks for reviewing I have made all the asked changes, plz check and merge the files